2022q4/cloud-init.adoc

```diff
@@ -1,7 +1,7 @@
 === FreeBSD as a Tier 1 cloud-init Platform
 
 Links: +
-https://https://cloud-init.io/[cloud-init Website] URL: https://cloud-init.io/[https://cloud-init.io/] +
+https://cloud-init.io/[cloud-init Website] URL: https://cloud-init.io/[https://cloud-init.io/] +
```

Comment:
That one's OK too.

Reply:
Do you mean, OK _without_ the `link:` prefix in nine places (in the _Links:_ section)?